pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.github.aquality-automation</groupId>
     <artifactId>aquality-selenium-core</artifactId>
-    <version>0.0.3</version>
+    <version>0.1.0</version>
```

Comment:
I beleive it is time to release major version 1.0.0

Reply:
I don't mind